### ISSUE TITLE
Update wiki to describe auto-linkification of fields with corresponding tags

### ISSUE THREAD

**Issue by bendotli** [koillection/koillection#840]
I noticed in many of the wiki screenshots, some fields were linkified, and I had a tough time figuring out how to get that happen on my instance.

Turns out you just need to create a tag that has the same name as your field value, and Koillection will auto-linkify the field value! It took a little bit of [digging in the code](https://github.com/benjaminjonard/koillection/blob/1.4/templates/App/Item/_content.html.twig#L97C49-L97C49) to learn about this feature, so it'd be great if it could be described on the [Items](https://github.com/benjaminjonard/koillection/wiki/Items) or [Tag](https://github.com/benjaminjonard/koillection/wiki/Tags) pages -- or both!

I'd make this wiki change myself but it seems you can't submit pull requests for GitHub wikis?

It's such a great feature, better than what I would have proposed (adding an option during template/item creation to mark a field as searchable, and making fields link to the generic search page or a new advanced search-by-field page for their value). Having tags means you can create them only for e.g. authors you really care about, and you can write a detailed description for that author!

**Comment by oywino** [comment]
What exactly  do you mean by _some fields were linkified_ ?
Do you mean that field name or content appeared clickable as a link?

**Comment by bendotli** [comment]
Text within field content that matches an existing tag becomes clickable as a link to that tag. The entire field content does not have to match, only a substring, and it is only that substring that becomes linkified. Multiple links within the same field content text are possible.

EDIT(1/22/2025): for future readers -- actually I misremembered; it looks like fields need exact match to auto-linkify, even if they are long text fields, but tag descriptions (and maybe elsewhere) exhibit the substring behavior (with the associated problems discussed below).

**Comment by oywino** [comment]
Can you give an example of some tags that would be suitable/usable?
I mean a tag named 'New' as in condition, could not be used as that word could easily also appear in a description field and in a very different context.  As such, how do you avoid complete chaos?

**Comment by bendotli** [comment]
I'm not requesting or advocating for this existing behavior, so it's not my responsibility to justify it. This issue I filed is merely requesting it not be left undocumented.

In my usage, I create tags for author names, which are unlikely to match on unrelated text. I agree that it leads to unreasonable behavior for tags that are common words, such as "New" as you mentioned. Perhaps you could file a separate issue for changes/improvements to this existing behavior (such as a settings option to toggle it off, or requiring special syntax such as `[New]` for linkification, both of which I would support).

**Comment by oywino** [comment]
No, not at all - and please don't get me wrong - I'm not in any way criticizing your point or objectives, quite the contrary, I'm just trying to understand how to take advantage of such a feature.
If, as you say - tags are used to store author names (which most likely are unique, I agree), then you'll end up with an awful lot of tags when your book database reaches 200.000 titles!
I wouldn't dream of using taks that way. In my mind, author's name is a field and not a tag.
But if you think I've misunderstood, please explain. 🙂
